bundles/binding/org.openhab.binding.dmx/src/main/java/org/openhab/binding/dmx/internal/core/DmxController.java

```diff
@@ -27,13 +27,15 @@
  * Default DMX Service implementation
  *
  * @author Davy Vanherbergen
+ * @author Jan N. Klug
  * @since 1.2.0
  */
 public class DmxController implements DmxService, ManagedService {
 
     private static Logger logger = LoggerFactory.getLogger(DmxController.class);
 
     private static int TRANSMIT_FREQUENCY_MS = 35;
+    private int repeatMode = DmxTransmitter.REPEAT_MODE_ALWAYS; // default is send every update
```

Comment:
Instead of `int`, it would be safer to use an `enum` type.  [Here](http://stackoverflow.com/a/2965252) is the construct that would seem to suit this optionally configured value.